ui/FilesPanel.go

```diff
@@ -90,32 +90,62 @@ func (this *filesPanel) createRefreshButton() *gtk.Button {
 
 func (this *filesPanel) createBackButton() *gtk.Button {
 	image := utils.MustImageFromFileWithSize("back.svg", this.Scaled(40), this.Scaled(40))
-	return utils.MustButton(image, func() {
-		if this.locationHistory.Length() < 1 {
-			this.UI.GoToPreviousPanel()
-		} else if this.locationHistory.IsRoot() {
-			this.locationHistory.GoBack()
-			this.doLoadFiles()
-		} else {
-			this.locationHistory.GoBack()
-			this.doLoadFiles()
-		}
-	})
+	return utils.MustButton(image, this.goBack)
 }
 
 func (this *filesPanel) doLoadFiles() {
 	utils.EmptyTheContainer(&this.listBox.Container)
-
-	if this.displayRootLocations() {
+	atRootLevel := this.displayRootLocations()
+	/**
```

Comment:
I'll just clean this up myself, but just a fyi, `/**` is a JSDoc comment (https://jsdoc.app/about-getting-started.html) and typically is only used when generating docs from code.

Reply:
As an FYI, it may be, but not exclusively:

https://pear.php.net/manual/en/standards.sample.php

That was just the first doc I could pull up, but there are plenty more. PSR-12, etc etc.

I've been programming for 15 years. It's a common enough comment style.

That said, if it goes against the codebase style, I will be sure to adjust when I contribute.

----

A quick look shows that the standard Go convention is:

```
// Single line comment

// Occasionally maybe
// multiple lines.

/*
C-Style block comments.

For when you really have 
to get your comment on.
*/
```

That about right? I'll sort it out next time around. My appologies.

Reply:
> As an FYI, it may be, but not exclusively:
> https://pear.php.net/manual/en/standards.sample.php
> That was just the first doc I could pull up, but there are plenty more. PSR-12, etc etc.

TL;DR - I was wrong.  

In several languages (eg JavaScript, Java, and C#, etc...) `/**` is used for auto-generating docs from the source code, and is supposed to be used only for the public-facing/consumable code (things you'd make an API guide from)

GoLang has GoDoc, and I thought it used the same syntax, however:
1) Searching the codebase, I see only a few instances of `/**`, but not everywhere (like you would expect, like in the libraries that are in vendor)
2) The docs for GoLang don't specify it, they say to just use `//`
https://blog.golang.org/godoc

I don't plan on generating and publish docs like this for OctoScreen, but I wanted to keep in line with what the GoLang community does.



> That said, if it goes against the codebase style, I will be sure to adjust when I contribute.

I was wrong.  I thought the doc-generating style was `/**`.  In the future feel free to use this style.  My concern had been that it would add internal comments to docs if they were generated, but that's not the case.



> I've been programming for 15 years. It's a common enough comment style.

Yea, I got the impression you've been programming for a while.  I can tell from your PRs that you're competent, but also impressed that you don't write "clever" code (meaning, some devs like to write complicated, obfuscated code, which no one can read, and a year later when the code needs to be enhanced, no one can change it w/o introducing bugs)

Reply:
Oh...  I'm only just barely breaking the habit of clever code. lol

I'll just stick to `//` and `/*` for GoLang.  In my experience, `/*` and `/**` have been relatively interchangeable, but it's really neither here nor there.  The best standard is a standard. :-D So I'll stick to the standard.